### PR TITLE
fix: add cache to 3idv0 resolver

### PR DESCRIPTION
**Pull request by oed**
Not urgent to review/merge.

### CODE REVIEW

File: packages/3id-did-resolver/src/legacyResolver.ts [hunk 1]
```diff
@@ -11,11 +12,17 @@ interface IPFS {
 
 // Legacy 3ids available from 3box, other v1 will always be resolved through ipfs and other services 
 const THREEBOX_API_URL = 'https://ipfs.3box.io'
+const LIMIT = 100
+const fetchCache = new LRUMap<string, any>(LIMIT)
 
 const fetchJson = async (url: string): Promise<any> =>  {
+  const cached = fetchCache.get(url)
+  if (cached) return cached
```
Comment (stbrody) [inline packages/3id-did-resolver/src/legacyResolver.ts:20]: can v0 3ids ever be updated?  Do we need to worry about cache invalidation at all?

Comment (ukstv) [reply packages/3id-did-resolver/src/legacyResolver.ts:20]: They can not. They are static, so no cache invalidation.

**Review by ukstv — APPROVED** [review]
🏇

File: packages/3id-did-resolver/src/legacyResolver.ts (hunk 1 above)
Comment (oed) [reply packages/3id-did-resolver/src/legacyResolver.ts:20]: Nope! 3IDv0 are just static ipfs objects.

**Pull request merged**